docker-compose.yaml

```diff
@@ -37,6 +37,9 @@ services:
         image: thecodingmachine/nodejs:v2-18-bullseye
         command: npm run dev
         working_dir: /usr/src/app/play
+        depends_on:
+            messages:
+                condition: service_healthy
```

Comment:
Not 100% sure about this.
Because the health check on "messages" won't be done before the start period (10 seconds), it means that the play container will take AT LEAST 10 more seconds to load. We are doing our best to speed up play by upgrading Vite, it would be a shame to slow it down.